docusaurus/docs/developer-docs/brizy-internals.md

```diff
@@ -3,3 +3,37 @@ sidebar_position: 2
 ---
 
 # Brizy Internals
+
+Brizy operates on a core set of technologies we call the internals. In this section, we'll guide you through the key concepts necessary for developing addons for Brizy.
+
+# Available "Internals"
+
+Below is a list of Brizy internals along with a short description and links to more in-depth information:
+
+[//]: # (TODO: Link for editor)
+- [Brizy Editor](/docs-internals/brizy-editor/introduction)<br/>
+The editor is where users create pages and control their settings. It is divided into preview and panel areas.
+The panel hosts the list of widgets and is where users control the settings. The preview screen is where they see how the page will look.
+
+[//]: # (TODO: Link for toolbar)
+- [Toolbar](/docs-internals/toolbar)<br/>
+The toolbar is used for adding options to each widget, allowing for easy customization and configuration.
+ 
+[//]: # (TODO: Link for sidebar)
+- [Sidebar](/docs-internals/sidebar)<br/>
+  There are two sidebars in Brizy: Left Sidebar and Right Sidebar. 
+  - The right sidebar allows users to set up effects and styling for widgets in the `Editor`.
+  - The left sidebar allows users to select widgets, apply global styling, change the view, and more.
```

Comment:
need to modify the description, because right sidebar also can be left sidebar:
![image](https://github.com/user-attachments/assets/6050892c-6c74-44f8-92d6-a50e0e7b8b1a)


need to write something more general as example that sidebar is the place where you can access the widgets or other controls